src/app/design/guidelines/page-layouts/index.html

```diff
@@ -1,21 +1,18 @@
 <stache
-  layout="container"
-  pageTitle="Page design"
+  pageTitle="Page Design"
+  navTitle="Page design"
   showTableOfContents="true">
 	
 	<stache-page-anchor>
-    Page <stache-code>&lt;title&gt;</stache-code>
+    Page title
   </stache-page-anchor>
 	
-	<p>Use the following format for the page's HTML <stache-code>&lt;title&gt;</stache-code> element:</p>
-	<p><span class="sky-emphasized">&lt;Page title&gt; - &lt;Product title&gt;</span></p>
-	<p>Example: <span class="sky-emphasized">Gifts list - Raiser's Edge NXT</span></p>
-	
-	<p>For pages with tabbed subnavigation, use the following format:</p>
-	<p><span class="sky-emphasized">&lt;Tab title&gt; - &lt;Page title&gt; - &lt;Product title&gt;</span></p>
-	<p>Example: <span class="sky-emphasized">Portfolio - Work center - Raiser's Edge NXT</span></p>
-	
-	<p>Additional page title guidelines:</p>
+  <p>
```

Comment:
This feels too detailed for a guideline page. Since the content is +/- duplicated from the app config page, can we instead put a short blurb/link at the end of the section instead of beginning saying something like "Refer to the app configuration documentation for information on how to implement page titles in SKYUX."

Reply:
I updated this based on your feedback. Please just let me know if you see anything else.